Gemfile

```diff
@@ -144,3 +144,9 @@ gem 'aws-sdk-core'
   # gem 'devise-guests', '~> 0.6'
 # end BL generated
 
+# we use for data structures for citation models, and for generating citations
+gem "citeproc-ruby", '~> 1.0'
+gem 'csl-styles', '~> 1.0' # Need to load the styles so we can use chicago
+# On MRI <= 2.3, citeproc-ruby insists upon `unicode` or `unicode_utils` gem. :(
+# https://github.com/inukshuk/citeproc/commit/c14d3cd272698dd4aa52625dd140864b7a7bd6cb
+gem 'unicode'
```

Comment:
we won't be using MRI <= 2.3, so shouldn't need this. Try it without it?  I don't think this particular PR is actually using citeproc-ruby or csl-styles at all yet? so we won't know if it works without it or not -- that's fine, we can leave it for the PR that uses it. so remove ALL of these, or just remove `unicode` now cause we don't think we need it.

Reply:
Oh, we do need citeproc-ruby to parse names, but can def. get rid of csl-styles for now. Unicode also not needed, as you noted.

Reply:
cool fine to leave csl-styles in, no big deal, we know we're gonna need it, the important thing is not adding dependencies we don't plan on needing!